src/lib/viewers/doc/DocFindBar.js

```diff
@@ -82,6 +82,7 @@ class DocFindBar extends EventEmitter {
         // Find input field
         this.findFieldEl = document.createElement('input');
         this.findFieldEl.classList.add('bp-doc-find-field');
+        this.findFieldEl.setAttribute('aria-label', 'Search input');
```

Comment:
Hi @fsuarez-globant, thanks for submitting this change. We require all our user-facing strings to support internationalization. Can you please add this string to `src/i18n/en-US.properties` and use it via the `__` helper?

Reply:
@jstoffan of course, I'll add this string in the internationalization file.